packages/client-sdk-nodejs/test/integration/integration-setup.ts

```diff
@@ -106,7 +124,7 @@ export function SetupAuthClientIntegrationTest(): {
 
   beforeAll(async () => {
     // Use a fresh client to avoid test interference with setup.
-    const momento = momentoClientForTesting();
+    const momento = momentoClientForTestingWithSessionToken();
```

Comment:
Uh, this would make *all* of the integration tests run using the session token, wouldn't it? I don't think we want that...

Reply:
Maybe I am misunderstanding - but that seems exactly what we want. The `momento` client should only be used for setting up the test resources, the actual tests themselves create different types of cache clients.

Reply:
whoops, my bad!

The way that github was showing me this diff, it looked like you were doing this in `beforeAll` for `SetupIntegrationTest`, which is the setup method that is used for ALL the cache integration tests.

After seeing @krispraws comment I realized there was an ellipsis there and that github was hiding a bunch of lines of code from me, and that this change is only happening in the `SetupAuthClientIntegrationTest` function, which is indeed only for auth tests, so this makes sense now. Thanks @krispraws